### ISSUE TITLE
Size of NCBI genome not checked in Downloader Lambda

### ISSUE THREAD

**Issue by DiegoQuintero1** [bmds-lab/Crackling-AWS#11]
The NCBI API was used for checking the genome filesize (FASTA file) prior to downloading. Though, the API was not consistent in its output, sometimes getting an unexpected value of 0 for a 2MB genome (resulting in the genome not being downloaded because of its unknown size). A 11GB file can also output 0 for its size (assumption). Thus, the NCBI api was removed in the cdk-dev version branch. 

For reference, there are 3750/ 74700 (5%) FASTA files from [https://www.ncbi.nlm.nih.gov/genome/browse#!/overview/](url) which exceed 650MB (the limit of Crackling pipeline - timeout errors otherwise). 

For this reason, determining the tradeoff for incorporating the NCBI API again must focused around cost, availability and security. 

- cost - downloading a FASTA file of 9-10GB incurs a cost of $0.45 in nat gateways, but file not used (**650MB limit**).
- availability - inability to download all files below 650MB because the tool is not consistent.
- security - financial risk, genome files >11GB files are able to start download, but the Downloader Lambda function will timeout (memory exceeded error).

NCBI API - located in **Scheduler Lambda function**:
![image](https://github.com/bmds-lab/Crackling-AWS/assets/62460407/3d1a7583-a38b-4c28-8000-7e8ae52d9942)

TLDR:
- Decide whether NCBI API should be added in again based on context (tradeoff).
- if yes, determine an approach for error handling (constraints) if tool is inconsistent. 
- if no, deal with security and cost accordingly. e.g. _Blacklist database of genome files > 650MB._

**Comment by jakeb1996** [comment]
The architecture for downloading genomes from NCBI has changed. This issue can be closed.
